validators.md

```diff
@@ -236,6 +236,14 @@ Peer | stellar0.keybase.io
 Node ID | GCWJKM4EGTGJUVSWUJDPCQEOEP5LHSOFKSA4HALBTOO4T4H3HCHOM6UX
 Status | ![Online](https://img.shields.io/badge/status-online-brightgreen.svg)
 
+Name | Sendgrow
+---|---
+Description | https://sendgrow.co/
+Contact | support@sendgrow.co
+Peer | validator.sendgrow.co
```

Comment:
[https://sendgrow.co/.well-known/stellar.toml](https://sendgrow.co/.well-known/stellar.toml) is not accessible. please can you fix.